newIDE/app/src/UI/Toggle.js

```diff
@@ -3,24 +3,52 @@ import * as React from 'react';
 import FormControlLabel from '@material-ui/core/FormControlLabel';
 import Switch from '@material-ui/core/Switch';
 import { makeStyles } from '@material-ui/core';
+import GDevelopThemeContext from './Theme/ThemeContext';
+import { type GDevelopTheme } from './Theme';
 
-const useStyles = makeStyles({
+const useFormStyles = makeStyles({
   root: {
     cursor: 'default',
   },
   labelPlacementStart: { marginLeft: 0 },
   label: { flex: 1 },
 });
 
+const useSwitchStyles = ({
+  theme,
+  toggled,
+  disabled,
+}: {
+  theme: GDevelopTheme,
+  toggled: boolean,
+  disabled: boolean,
+}) =>
+  makeStyles({
+    thumb: {
+      backgroundColor: disabled
+        ? theme.switch.thumbColor.disabled
+        : toggled
+        ? theme.switch.thumbColor.toggled
+        : theme.switch.thumbColor.default,
+    },
+    track: {
+      backgroundColor: disabled
+        ? theme.switch.trackColor.disabled
+        : toggled
+        ? theme.switch.trackColor.toggled
+        : theme.switch.trackColor.default,
+      opacity: '1 !important', // MUI overrides the opacity to 0.5, prevent this.
+    },
+  })();
+
 // We support a subset of the props supported by Material-UI v0.x Toggle
 // They should be self descriptive - refer to Material UI docs otherwise.
 type Props = {|
-  label?: ?React.Node,
+  label: React.Node,
   toggled: boolean,
   onToggle: (e: {||}, toggled: boolean) => void,
   disabled?: boolean,
   labelPosition: 'right' | 'left',
-  size?: 'small',
```

Comment:
this prop wasn't doing anything on the component + used only in 1 place, I checked and didn't see any change